mediacapture-record/idlharness.window.js

```diff
@@ -0,0 +1,39 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/mediacapture-record/
+
+idl_test(
+  ['mediastream-recording'],
+  ['mediacapture-main', 'html', 'dom', 'FileAPI'],
```

Comment:
Needed?

Reply:
Yep - `Blob` is used by `BlobEventInit`.